test/AutoDiff/validation-test/property_wrappers.swift

```diff
@@ -131,16 +126,18 @@ PropertyWrapperTests.test("SimpleClass") {
               gradient(at: Class(), 2, of: setter))
   */
 
-  // TODO: Support `modify` accessors (https://github.com/apple/swift/issues/55084).
-  /*
+  // FIXME(TF-1175): Same issue as above
```

Comment:
This seems like an accidental change? Since the TF issues references no longer exist right?

Reply:
No, it was intentional. The test in question is about class values differentiation (and corresponding property wrappers). Before we issued an error and now we produce an incorrect answer exactly like the case above. `TF-1175` is a marker used around codebase for the class-valued differentiation, so I used it to mark the case.

Reply:
Ah ok my bad